src/content.rs

```diff
@@ -167,6 +169,20 @@ pub fn all_pages(
     }
 }
 
+pub fn get_pages_by_glob(
+    dir: PathBuf,
+    globs: &Vec<String>,
+    show_unpublished: bool,
+) -> anyhow::Result<BTreeMap<String, PageValues>> {
+    let index_cache: IndexCache = pages_by_glob_load(dir, globs, show_unpublished)?;
+    Ok(index_cache.contents)
+}
+
+pub fn all_pages_load(dir: PathBuf, show_unpublished: bool) -> anyhow::Result<IndexCache> {
+    let files = all_files(dir)?;
+    let index_cache = get_index_cache_from_files(files, show_unpublished)?;
```

Comment:
nit: I think this and the next line could be condensed down to

```suggestion
    get_index_cache_from_files(files, show_unpublished)
```